frontend/web/components/OrganisationSelect.js

```diff
@@ -8,6 +8,12 @@ const OrganisationSelect = class extends Component {
     this.state = {}
   }
 
+  componentDidMount() {
+    if (this.props.firstOrganisation) {
+      this.props.onChange(AccountStore.getOrganisation().id)
```

Comment:
This will probably crash the app if the org hasn't loaded yet

Reply:
Corrected.